module-4/backend/week_two.md

```diff
@@ -0,0 +1,56 @@
+## Week Two - Module 4 Recap
+
+
+1. What is Webpack and why is it useful?
+
+- It's essentially a bundler for JS. Allows you to pull many different resources together under one, or multiple codebases.
+
+2. When do you want to use event delegation?
```

Comment:
this is why sometimes we need to add an event listener to a parent rather directly on the child - i think this may have been why you had trouble with the delete events!